README.rst

```diff
@@ -18,6 +18,15 @@ without any arguments:
 .. code-block:: bash
 
     $ php sami.phar
+    
+*Note:* Sami 4.0.x requires PHP 7 to work correctly; Sami 3.3.0 can be run with PHP 5.3.9 or higher. 
+
+Install with composer:
```

Comment:
This should be removed. It's highly not recommended to install Sami with Composer.